client/app/modules/tests-sessions/test-session-card/test-session-card.scss

```diff
@@ -0,0 +1,322 @@
+@import "../../../../styles/global/var";
+@import "../../../../styles/ui/components/animation";
+
+.test-session-card {
+    &__clickable {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        left: 0;
+        z-index: 1;
+        &:hover {
+            background-color: rgba(0,0,0,.05);
+        }
+        &._highlighting {
+            animation: change-color 3s ease-in-out 1;
+        }
+    }
+    &__wrapper {
+        position: relative;
+        background-color: #fff;
+        border-left: 4px solid #777;
+        min-height: 55px;
+        display: flex;
+
+        &-left,
+        &-right {
+            display: flex;
+        }
+        &-left {
+            width: 40%;
+            flex-grow: 1;
+            min-width: 400px;
+        }
+        &-right {
+            width: 60%;
+            position: relative;
+        }
+        &.COMPLETED {
```

Comment:
I think it's better to use `lowercase` pipe in the html and use lowercase for class name

Reply:
fixed